art/wrappers/randomized_smoothing.py

```diff
@@ -0,0 +1,226 @@
+# MIT License
+#
+# Copyright (C) IBM Corporation 2018
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+# persons to whom the Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
+# Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+# WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""
+This module implements Randomized Smoothing applied to classifier predictions.
+
+Paper link:
+    https://arxiv.org/pdf/1902.02918.pdf
+
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+
+from art.wrappers.wrapper import ClassifierWrapper
```

Comment:
Could you please reorder the imports in lines 29-33 to list third party imports before local application imports.
```suggestion
import numpy as np
from scipy.stats import norm, binom_test

from art.wrappers.wrapper import ClassifierWrapper
```